package.json

```diff
@@ -38,8 +38,6 @@
     "mappy-breakpoints": "^0.2.3",
     "redis": "^2.8.0",
     "susy": "^3.0.5",
-    "unzip": "^0.1.11",
```

Comment:
This dependency was never used, not even in the initial commit where it was added.

Reply:
Agreed, hence it needs to go :)